tsconfig.json

```diff
@@ -7,11 +7,12 @@
     "esModuleInterop": true,
     "strict": true,
     "strictNullChecks": true,
+    "noUncheckedIndexedAccess": true,
```

Comment:
A bit stricter typescript checking. I had to disable this for `webcert` and `rehabstöd` since it would cause a lot of ts issues atm.